### PR TITLE
Update pt.json

### PR DESCRIPTION
**Pull request by adberaldo**
Update translation PT-BR

#### How to test it?
Navigate the app to check messages

[Workspace](Link goes here!)

### CODE REVIEW

File: messages/pt.json [hunk 1]
```diff
@@ -1,86 +1,67 @@
...
-  "store/b2b-quotes.quotes-table.filters.any": "Any",
-  "store/b2b-quotes.quotes-table.filters.is": "is",
-  "store/b2b-quotes.quotes-table.filters.includes": "includes",
-  "store/b2b-quotes.quote-details.title": "Quote Details",
-  "store/b2b-quotes.quote-details.decline": "Decline",
-  "store/b2b-quotes.quote-details.save": "Save Quote",
-  "store/b2b-quotes.quote-details.submit-to-sales-rep": "Submit to Sales Rep",
-  "store/b2b-quotes.quote-details.use-quote": "Use Quote",
+  "store/b2b-quotes.quotes-table.filters.all": "Tudo",
+  "store/b2b-quotes.quotes-table.filters.none": "Nada",
```
Comment (arturmagalhaesjr) [inline messages/pt.json:27]: It could be "Nenhum" instead of "Nada"

File: messages/pt.json [hunk 2]
```diff
@@ -1,86 +1,67 @@
...
+  "store/b2b-quotes.quotes-table.filters.any": "Qualquer",
+  "store/b2b-quotes.quotes-table.filters.is": "é",
+  "store/b2b-quotes.quotes-table.filters.includes": "inclui",
+  "store/b2b-quotes.quote-details.title": "Detalhes da Cotação",
+  "store/b2b-quotes.quote-details.decline": "Reprovar",
+  "store/b2b-quotes.quote-details.save": "Salvar Cotação",
+  "store/b2b-quotes.quote-details.submit-to-sales-rep": "Enviar ao Vendedor",
```
Comment (arturmagalhaesjr) [inline messages/pt.json:34]: "Representante de Vendas"

File: messages/pt.json [hunk 3]
```diff
@@ -1,86 +1,67 @@
...
-  "store/b2b-quotes.quote-details.expiration.title": "Expiration",
+  "store/b2b-quotes.quote-details.expiration.title": "Expiração",
   "store/b2b-quotes.quote-details.status.title": "Status",
-  "store/b2b-quotes.quote-details.items.image.title": "Image",
-  "store/b2b-quotes.quote-details.items.refId.title": "Ref. Code",
-  "store/b2b-quotes.quote-details.items.name.title": "Name",
-  "store/b2b-quotes.quote-details.items.quantity.title": "Quantity",
-  "store/b2b-quotes.quote-details.items.price.title": "Price",
+  "store/b2b-quotes.quote-details.items.image.title": "Imagem",
+  "store/b2b-quotes.quote-details.items.refId.title": "Cód.Refer.",
```
Comment (arturmagalhaesjr) [inline messages/pt.json:40]: "Código Ref "

**Review by arturmagalhaesjr — CHANGES_REQUESTED** [review]

**Comment by enzomerca** [comment]
This is an old PR and these translations are already fixed on `master`

**Pull request closed without merging**